apps/odo/runtime.exs

```diff
@@ -32,7 +32,6 @@ config :ex_aws,
   region: System.get_env("AWS_REGION") || "us-west-2"
 
 config :odo, :brook,
-  instance: :odo_brook,
```

Comment:
are we positive brook still is able to receive events with this change? Looking at `application.ex` it's unclear how it would work.